pages/common/nc.md

```diff
@@ -30,10 +30,10 @@
 
 `nc -q {{timeout}} {{ip_address}}`
 
-- Port scanning:
+- Scan the open ports of a specified host:
 
 `nc -v -z {{ip_address}} {{port}}`
 
-- Proxy and port forwarding:
+- Proxy and port forwarding, forward data from a local TCP port to the given remote host:
```

Comment:
Maybe we can remove "port forwarding" too just as we did in the earlier example. So perhaps - "Act as proxy and forward data from ..."

Reply:
I agree, looks cleaner.

Reply:
Updated.